File: newActiveStates.py
#! C:/Users/P/anaconda3/python
print('Content-type: text/html; charset=utf-8\n\n')
import sqlite3
con = sqlite3.connect('fsmx.db')
cur = con.cursor()
cur.execute('delete from activeStates')
cur.execute('insert into activeStates(activeState) values (1)')
con.commit()

File: newActiveStates.php
<?php
$db = new SQLite3('fsmx.db');
$db->query('delete from activeStates');
$db->query('insert into activeStates(activeState) values (20)'); 
$addr=$_SERVER['SERVER_ADDR'];
header('Location: http://'.$addr.'/fsmx.php');
//header('Location: 192.168.1.163/fsmx.php');
?>
